webapi/src/main/scala/org/knora/webapi/util/DateUtilV1.scala

```diff
@@ -96,7 +96,24 @@ object DateUtilV1 {
         val daysInMonth = Calendar.DAY_OF_MONTH // will be used to determine the number of days in the given month
         // val monthsInYear = Calendar.MONTH // will be used to determine the number of months in the given year (generic for other calendars)
 
-        val dateSegments = dateString.split(InputValidation.PrecisionSeparator)
+        var era: Option[Int] = None
```

Comment:
the use of `var` should be avoided. Can't this be rewritten?

For instance you could make a `match` statement return a value and assign that to a variable declared with `val`.